docs/advanced_topics/audit_log.rst

```diff
@@ -71,6 +71,7 @@ Action                               Notes
 ``wagtail.rename``                   A page was renamed
 ``wagtail.revert``                   The page was reverted to a previous draft
 ``wagtail.copy``                     The page was copied to a new location
+``wagtail.copy_for_translation``     The page was copied into a new locale for translation
```

Comment:
Just a thought... any reason not to follow the hierarchical pattern of the other action names, and make it something like `wagtail.copy.for_translation`?

Reply:
It seems a bit inconsistent the way it's done already. For example `wagtail.revert` is a type of edit, and ``wagtail.publish.schedule`` is not publishing the page, just scheduling it